client/blocks/plan-thank-you-card/docs/example.jsx

```diff
@@ -24,7 +24,11 @@ function PlanThankYouCardExample( { primarySiteId } ) {
 }
 
 const ConnectedPlanThankYouCard = connect( ( state ) => {
-	const primarySiteId = get( getCurrentUser( state ), 'primary_blog' );
```

Comment:
Adding a `false` as third argument to `get()` should work here. I don't believe it's any less confusing than setting `primarySiteId` to `-1`